docs/docs/mlflow-3/index.mdx

```diff
@@ -0,0 +1,348 @@
+            mlflow.log_metric(
+                key="accuracy",
+                value=compute_accuracy(scripted_model, X_train, y_train),
+                step=epoch,
+                model_id=model_info.model_id,
+                dataset=train_dataset,
+            )
+
+ranked_checkpoints = mlflow.search_logged_models(
+    filter_string=f"source_run_id='{run.info.run_id}'",
+    order_by=[{"field_name": "metrics.accuracy", "ascending": False}],
+    output_format="list",
+)
+
+best_checkpoint = ranked_checkpoints[0]
+print(f"Best model: {best_checkpoint}")
+print(best_checkpoint.metrics)
+
+# Best model: <LoggedModel: artifact_location='file:///Users/serena.ruan/Documents/repos/mlflow/mlruns/0/models/4661d15d-84ae-4896-b9cf-3731be3fac7b/artifacts', creation_timestamp=1742784533012, experiment_id='0', last_updated_timestamp=1742784538509, metrics=[<Metric: dataset_digest='1f1c13b5', dataset_name='train', key='accuracy', model_id='4661d15d-84ae-4896-b9cf-3731be3fac7b', run_id='92b65129f25a412f9455e637a3d95a2a', step=100, timestamp=1742784538522, value=0.9833333333333333>], model_id='4661d15d-84ae-4896-b9cf-3731be3fac7b', model_type=None, model_uri='models:/4661d15d-84ae-4896-b9cf-3731be3fac7b', name='torch-iris-100', params={'activation': 'ReLU',
+#  'criterion': 'CrossEntropyLoss',
+#  'n_layers': '3',
+#  'optimizer': 'Adam'}, source_run_id='92b65129f25a412f9455e637a3d95a2a', status=<LoggedModelStatus.READY: 'READY'>, status_message=None, tags={}>
+# [<Metric: dataset_digest='1f1c13b5', dataset_name='train', key='accuracy', model_id='4661d15d-84ae-4896-b9cf-3731be3fac7b', run_id='92b65129f25a412f9455e637a3d95a2a', step=100, timestamp=1742784538522, value=0.9833333333333333>]
+
+worst_checkpoint = ranked_checkpoints[-1]
+print(f"Worst model: {worst_checkpoint}")
+print(worst_checkpoint.metrics)
+
+# Worst model: <LoggedModel: artifact_location='file:///Users/serena.ruan/Documents/repos/mlflow/mlruns/0/models/da03f02f-1352-456a-97dc-3cf4f1f83fae/artifacts', creation_timestamp=1742784476515, experiment_id='0', last_updated_timestamp=1742784483429, metrics=[<Metric: dataset_digest='1f1c13b5', dataset_name='train', key='accuracy', model_id='da03f02f-1352-456a-97dc-3cf4f1f83fae', run_id='92b65129f25a412f9455e637a3d95a2a', step=0, timestamp=1742784483447, value=0.3333333333333333>], model_id='da03f02f-1352-456a-97dc-3cf4f1f83fae', model_type=None, model_uri='models:/da03f02f-1352-456a-97dc-3cf4f1f83fae', name='torch-iris-0', params={'activation': 'ReLU',
+# 'criterion': 'CrossEntropyLoss',
+# 'n_layers': '3',
+# 'optimizer': 'Adam'}, source_run_id='92b65129f25a412f9455e637a3d95a2a', status=<LoggedModelStatus.READY: 'READY'>, status_message=None, tags={}>
+# [<Metric: dataset_digest='1f1c13b5', dataset_name='train', key='accuracy', model_id='da03f02f-1352-456a-97dc-3cf4f1f83fae', run_id='92b65129f25a412f9455e637a3d95a2a', step=0, timestamp=1742784483447, value=0.3333333333333333>]
+```
+
+
+### GenAI Example
```

Comment:
Can we create separate pages for each example, and link them in `index.mdx`?

Reply:
I would prefer to do so later when we have UI to explain the lineage, right now it's quite short to fit in a separate page 🤔

Reply:
Makes sense :)